src/Backend/Utils.js

```diff
@@ -0,0 +1,11 @@
+export function uuidv4() {
+    return ([1e7] + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, c =>
+        (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
+    );
+}
```

Comment:
Code copied from StackOverflow should be correctly cited to maintain authors copyrights
See this old post https://stackoverflow.blog/2009/06/25/attribution-required/
and some newer entries https://meta.stackexchange.com/questions/25956/what-is-up-with-the-source-code-license-on-stack-overflow/25957#25957

```suggestion
// copied from https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid/2117523#2117523
export function uuidv4() {
    return ([1e7] + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, c =>
        (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
    );
}
```